Model/Sales/Repository/PackageRepository.php

```diff
@@ -168,7 +195,8 @@ public function setMailboxSettings()
 
         $this->setMailboxActive($settings['active'] === '1');
         if ($this->isMailboxActive() === true) {
-            $this->setMaxWeight((int) $settings['weight'] ?: self::DEFAULT_MAILBOX_WEIGHT);
+            $weight = str_replace(',', '.', $settings['weight']);
+            $this->setMaxMailboxWeight($weight ?: self::MAXIMUM_MAILBOX_WEIGHT);
```

Comment:
is `self::MAXIMUM_MAILBOX_WEIGHT` hernoemen naar `self::DEFAULT_MAXIMUM_MAILBOX_WEIGHT` niet beter?